docs/UserGuide.md

```diff
@@ -334,8 +334,19 @@ Examples:
 
 ![view](images/viewWarehouse.png)
 
+### Undo/Redo a previous editing : `undo`/`redo`
+
+Recovers the previous version before an editing on the CLI-nic data or restores the later version of edited data after
+a recovery to previous version.
```

Comment:
This sounds a bit confusing. Maybe:
Recovers a previous version of CLI-nic data if any data has been changed by using undo command. Redo command will restores the data in CLI-nic before an undo command was done.

Reply:
Thanks, I'm thinking "Undo command recovers a previous version of CLI-nic data if any data has been changed. Redo command will restores the data in CLI-nic before an undo command was done."